tools/block-generator/generator/config.go

```diff
@@ -81,9 +81,6 @@ const (
 	assetTotal uint64 = 100_000_000_000_000_000 // 100 billion units per asset
 
 	consensusTimeMilli int64 = 3300
-
-	// TODO: do we still need this as can get it from the Ledger?
-	startingTxnCounter uint64 = 1000
```

Comment:
As hinted in the `TODO` comment, we can get it from the ledger's genesis block so we no longer require this hard-coded constant.